src/controllers/roverController.js

```diff
@@ -0,0 +1,27 @@
+const { getRoverImage } = require('../useCases/roverUseCase');
+
+const getRoverImageForUser = async (req, res) => {
+    try {
+        const { userId, userName, userApiKey } = req.body;
+
+        if (!userId || !userName || !userApiKey) {
+            return res.status(400).json({ error: 'Missing required user data' });
```

Comment:
let's create custom Exception with status code and error message
If any difficulties arise, you can always look at how other guys wrote it :)